client/my-sites/plugins/plugin-autoupdate-toggle/index.jsx

```diff
@@ -113,43 +113,54 @@ class PluginAutopdateToggle extends React.Component {
 	}
 
 	render() {
-		if ( ! this.props.site.jetpack ) {
+		const { site, plugin, translate, disabled } = this.props;
+		if ( ! site.jetpack ) {
 			return null;
 		}
 
-		const inProgress = PluginsLog.isInProgressAction( this.props.site.ID, this.props.plugin.slug, [
+		const inProgress = PluginsLog.isInProgressAction( site.ID, plugin.slug, [
 				'ENABLE_AUTOUPDATE_PLUGIN',
 				'DISABLE_AUTOUPDATE_PLUGIN'
 			] ),
 			getDisabledInfo = this.getDisabledInfo(),
 			label = getDisabledInfo
-				? i18n.translate( 'Autoupdates disabled', {
+				? translate( 'Autoupdates disabled', {
 					context: 'this goes next to an icon that displays if the plugin has "autoupdates disabled" active'
 				} )
-				: i18n.translate( 'Autoupdates', {
+				: translate( 'Autoupdates', {
 					context: 'this goes next to an icon that displays if the plugin has "autoupdates" active'
 				} );
 
 		return (
 			<PluginAction
-				disabled={ this.props.disabled }
+				disabled={ disabled }
 				label={ label }
-				status={ this.props.plugin.autoupdate }
-				action={ this.toggleAutoupdates }
+				status={ plugin.autoupdate }
+				action={ this.toggleAutoUpdates }
 				inProgress={ inProgress }
 				disabledInfo={ getDisabledInfo }
-				htmlFor={ 'autoupdates-' + this.props.plugin.slug + '-' + this.props.site.ID } />
+				htmlFor={ 'autoupdates-' + plugin.slug + '-' + site.ID } />
 		);
 	}
 }
 
-PluginAutopdateToggle.displayName = 'PluginAutopdateToggle';
+PluginAutoUpdateToggle.displayName = 'PluginAutoUpdateToggle';
 
-PluginAutopdateToggle.propTypes = {
-	isMock: React.PropTypes.bool,
-	site: React.PropTypes.object.isRequired,
-	plugin: React.PropTypes.object.isRequired,
-	wporg: React.PropTypes.bool
+PluginAutoUpdateToggle.propTypes = {
+	isMock: PropTypes.bool,
+	site: PropTypes.object.isRequired,
+	plugin: PropTypes.object.isRequired,
+	wporg: PropTypes.bool
 };
 
-export default PluginAutopdateToggle;
+PluginAutoUpdateToggle.defaultProps = {
+	isMock: false,
+	disabled: false,
```

Comment:
What do you think about including the `disabled` prop in the list of `propTypes`? It doesn't make sense to have a default value for it, but not have it listed as an expected prop.